app/views/users/show.html.erb

```diff
@@ -143,6 +143,16 @@
       <div class='text-body-secondary'>
         <small>
           <dl class="list-inline">
+            <% if @user.home_location_name&.strip.present? %>
+              <dt class="list-inline-item m-0">
+                <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-geo-alt-fill" viewBox="0 0 16 16">
```

Comment:
I think I'd be inclined to add `align-bottom` to the classes here - it seems to give slightly better vertical centring of the icon relative to the text for me?